dao/src/main/java/com/anna/config/DaoConfig.java

```diff
@@ -66,12 +83,18 @@ public static PropertySourcesPlaceholderConfigurer propertyConfigurer() {
   public DataSource dataSource() {
     DriverManagerDataSource dataSource = new DriverManagerDataSource();
     dataSource.setDriverClassName(driverClassName);
+    System.out.println(driverClassName);
     dataSource.setUrl(url);
     dataSource.setUsername(name);
     dataSource.setPassword(password);
 
-    Resource initSchema = new ClassPathResource("db.sql");
-    Resource textSchema = new ClassPathResource("db_text.sql");
```

Comment:
We can use String.Format to get the strings "conf.environment/%s/db.sql" with appropriate db name
And we can use Map with key =  org.postgresql.Driver and value POSTGRESQL